editor/src/components/editor/import-wizard/components.tsx

```diff
@@ -0,0 +1,320 @@
+      return 0
+    }
+    if (operation.timeDone == null) {
+      return currentTime - operation.timeStarted
+    }
+    return operation.timeDone - operation.timeStarted
+  }, [operation.timeStarted, operation.timeDone, currentTime])
+  const timeInSeconds =
+    operation.timeDone != null
+      ? (operationTime / 1000).toFixed(2)
+      : Math.max(Math.floor(operationTime / 1000), 0)
+  return operation.timeStarted == null ? null : (
+    <div
+      data-short-time={operationTime < 100}
+      style={{
+        color: runningStatus === 'running' ? 'black' : 'gray',
+        fontSize: runningStatus === 'running' ? undefined : 12,
+      }}
+    >
+      {timeInSeconds}s
+    </div>
+  )
+}
+
+function OperationLineWrapper({
+  children,
+  className,
+  onClick,
+}: {
+  children: React.ReactNode
+  className: string
+  onClick?: () => void
+}) {
+  return (
+    <div
+      className={`import-wizard-operation-line ${className}`}
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        gap: 10,
+      }}
+      css={{
+        '.import-wizard-operation-children > &': {
+          paddingLeft: 26,
+          fontSize: 12,
+          img: {
+            width: 12,
+            height: 12,
+          },
+        },
+        '.import-wizard-operation-children .operation-done [data-short-time=true]': {
+          visibility: 'hidden',
+        },
+      }}
+      onClick={onClick}
+    >
+      {children}
+    </div>
+  )
+}
+
+function OperationLineContent({
+  children,
+  textColor,
+}: {
+  children: React.ReactNode
+  textColor: string
+}) {
+  return (
+    <div
+      className='import-wizard-operation-line-content'
+      style={{
+        display: 'grid',
+        gridTemplateColumns: '15px max-content 1fr 14px',
+        gap: 10,
+        alignItems: 'center',
+        color: textColor,
+      }}
+    >
+      {children}
+    </div>
+  )
+}
+
+function getImportOperationText(operation: ImportOperation): React.ReactNode {
+  if (operation.text != null) {
+    return operation.text
+  }
+  switch (operation.type) {
+    case 'loadBranch':
+      return (
+        <span>
+          Loading branch{' '}
+          <strong>
+            {operation.githubRepo?.owner}/{operation.githubRepo?.repository}@{operation.branchName}
+          </strong>
+        </span>
+      )
+    case 'fetchDependency':
+      return `Fetching ${operation.dependencyName}@${operation.dependencyVersion}`
+    case 'parseFiles':
+      return 'Parsing files'
+    case 'refreshDependencies':
+      return 'Fetching dependencies'
+    case 'checkRequirements':
+      return 'Checking Utopia requirements'
+    case 'checkRequirementAndFix':
+      return operation.text
+    default:
+      assertNever(operation)
+  }
+}
+
+function getTextColor(
+  operationRunningStatus: 'waiting' | 'running' | 'done',
+  operation: ImportOperation,
+) {
+  if (operationRunningStatus === 'waiting') {
+    return 'gray'
```

Comment:
Should this be using our color themes? As I would surmise that it will look a little weird in dark mode.

Reply:
yes, definitely should come from the theme... this is also temporary design and definitely going to be changed in the design iteration